vars/mavenExecute.groovy

```diff
@@ -36,31 +36,37 @@ import static com.sap.piper.Utils.downloadSettingsFromUrl
      * Configures maven to log successful downloads. This is set to `false` by default to reduce the noise in build logs.
      * @possibleValues `true`, `false`
      */
-    'logSuccessfulMavenTransfers'
+    'logSuccessfulMavenTransfers',
+    /**
+     * Returns the standard output of the maven command for further processing. By default stdout will be printed to the console but not returned by mavenExecute.
```

Comment:
That sound bogus. From the comment here I have the impression that this flag controls to where the output of the maven command gets delivered (`stdout` or somewhere else). But below we see `returnStdout: true` is currently hard coded. So this flag does not control that. Currently the maven output is always returned to a variable names `commandOutput` and we have to possibility to deliver it to `stdout`.

With that flag we control if the maven output is returned from the step call as a return value. It is an open question what we get returned in case we set `returnStdout` to `false`.